doc/manual.rst

```diff
@@ -71,16 +74,18 @@ Turning off preprocessing
 Preprocessing in *auto-sklearn* is divided into data preprocessing and
 feature preprocessing. Data preprocessing includes One-Hot encoding of
 categorical features, imputation of missing values and the normalization of
-features or samples. These steps currently cannot be turned off. Feature
-preprocessing is a single transformer which implements for example feature
+features or samples. Dataprerocessing steps cannot be turned off as this ensures
```

Comment:
Wasn't that a feature that was implemented by @rabsr in #977? Maybe we need to extend the documentation on this?

Reply:
Yes it needs to be documented properly, this was just a quick fix to give some space and clearer reference to what's being talked about rather than 'these'.

I believe this should be properly handled in the issue for addressing the walkthrough manual